packages/datatrak-web-server/src/app/middleware/attachAccessPolicy.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
```

Comment:
Time to update this snippet? I thought it was smart enough to just use the current year 🤔

```suggestion
 * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
```

<details>
<summary>Not-very-relevant nitpick 😅</summary>
My inner typographer also wants to nitpick that we should have `Copyright` or `©`, but not both, since this is like saying “copyright copyright”, but we have so many instances it isn’t worth amending</details>